pages/about.tsx

```diff
@@ -351,15 +373,19 @@ const AboutPage = () => {
 
       <SectionWrapper header={<H3>Access Sets</H3>} anchorKey="accesssets">
         <p className="pb-6">
-          Access sets are defined per transaction, and not per call. Access sets
-          can be thought of as two independent types of lists: those of touch
-          contract addresses, and those of touched contract storage slots.
+          Access sets are defined per transaction, and not per call. Each
+          transaction may be defined by some combination of its sender,
+          calldata, or callee. Transactions can either be external or internal.
+          External transactions are sent to the Ethereum network. Internal
+          transactions are external transactions that have executed the xCALLx
```

Comment:
My mistake:
```suggestion
          transactions are external transactions that have executed the xCALL
```

Reply:
👍

Reply:
You put the thumbs up but I don't see the change, so what does it mean?

Reply:
I'm not certain why these didn't go through, but I added them in manually. All that should be left is my one grammar question.